### PR TITLE
[v7r3] Added general service configuration options

### LINKED ISSUE #4077: extensions should be able to define their own Properties
Currently, the Properties are all hardcoded in `Core/Security/Properties.py` with no possibility for the extension to add one. 
@marianne013 would love that

### PR DESCRIPTION
**Pull request by atsareg**
This PR adds general service configuration options description including the Authorization section options. Fixes #4077 .

BEGINRELEASENOTES

*docs
NEW: Added general service configuration options description including Authorization options

ENDRELEASENOTES

### CODE REVIEW

File: docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst [hunk 1]
```diff
@@ -6,9 +6,89 @@ General Component Configuration Options
 
 This sections contains the configuration options that all services, agents, or executors share.
 
+
+* Module: this is the name of the component class to be instantiated. If a component with this name
+  also exists in the configuration, its options are inherited by the derived component
+* LogLevel: possible values INFO,VERBOSE,DEBUG
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst:12]:
```suggestion
* LogLevel: possible values ERROR,INFO,NOTICE,VERBOSE,DEBUG
```
?

File: docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst [hunk 2]
```diff
@@ -6,9 +6,89 @@ General Component Configuration Options
...
+            MaxWaitingPetitions = 250
+            Authorization
+            {
+              # By default only users of the VO biomed can access all the methods except see below
+              Default = vo:biomed
+              # Only members of the biomed_data_admin can use this method
+              eraseAllData = group:biomed_data_admin
+              # Only users from VO biomed and belonging to groups with NormalBiomedUser
+              # property can use this method
+              getReplicas = vo:biomed, NormalBiomedUser
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst:89]: Can you add above that the properties are AND'ed?

File: docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst [hunk 3]
```diff
@@ -6,9 +6,89 @@ General Component Configuration Options
...
+
+Group properties are defined in the Group definition in the Registry section as
+Properties option which accepts comma separated list of strings. These string values,
+Group properties, can be used in the Method access rules. Group properties are not
+limited to some predefined set of values. However, some common values are defined
+in the :py:mod:`~DIRAC.Core.Security.Properties` as string constants that can be
+used in the code. Use those values unless you need some more specific ones.
+
+Group name can be specified directly in the method access rules as a property of the form
+group:<group_name> .
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst:46]:
```suggestion
``group:<group_name>``.
```

File: docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst [hunk 4]
```diff
@@ -6,9 +6,89 @@ General Component Configuration Options
...
+
+VO name can be also specified directly as a property of the form vo:<vo_name> .
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst:48]:
```suggestion
VO name can be also specified directly as a property of the form ``vo:<vo_name>``.
```

File: docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst [hunk 5]
```diff
@@ -6,9 +6,89 @@ General Component Configuration Options
...
+* Default: default access rules for all the methods not specified explicitly
+* <Method>: access rules for a specific method
+
+Access rules are defined in terms of user Group properties, group name or VO name.
+
+Group properties are defined in the Group definition in the Registry section as
+Properties option which accepts comma separated list of strings. These string values,
+Group properties, can be used in the Method access rules. Group properties are not
+limited to some predefined set of values. However, some common values are defined
+in the :py:mod:`~DIRAC.Core.Security.Properties` as string constants that can be
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst:42]:
```suggestion
in the :py:mod:`~DIRAC.Core.Security.Properties` modules as string constants that can be
```

File: docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst [hunk 6]
```diff
@@ -6,9 +6,89 @@ General Component Configuration Options
...
+used in the code. Use those values unless you need some more specific ones.
+
+Group name can be specified directly in the method access rules as a property of the form
+group:<group_name> .
+
+VO name can be also specified directly as a property of the form vo:<vo_name> .
+
+Other possible values are:
+
+* `all` or `any`: for any user
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst:52]:
```suggestion
* ``all`` or ``any`` for any user
```

File: docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst [hunk 7]
```diff
@@ -6,9 +6,89 @@ General Component Configuration Options
...
+* `authenticated` for registered users
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst:53]:
```suggestion
* ``authenticated`` for registered users
```

File: docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst (hunk 1 above)
Comment (fstagni) [reply docs/source/AdministratorGuide/Configuration/generalConfigOptions.rst]: ERROR is used a lot.

**Pull request merged**